bot/main.py

```diff
@@ -45,12 +53,22 @@
 MONGO_DATABASE = os.getenv("MONGO_DATABASE")
 MONGO_URI = os.getenv("MONGO_URI")
 BASE_URL = os.getenv("BASE_URL")
+BOT_ADMINS = os.getenv("BOT_ADMINS").split(",")
```

Comment:
Just a preference, I'd like to store this as a list of ints instead. So that checks are more convenient and we don't have to convert id to str everytime

Reply:
Sure, I'll add that.